crates/net/README.md

```diff
@@ -19,7 +19,7 @@
 <sub>Built with 🦀🕸 by <a href="https://rustwasm.github.io/">The Rust and WebAssembly Working Group</a></sub>
 </div>
 
-HTTP requests library for WASM Apps. It provides idiomatic Rust bindings for the `web_sys` `fetch` and `WebSocket` API
+HTTP requests library for WASM Apps. It provides idiomatic Rust bindings for the `web_sys` `EventSource`, `fetch` and `WebSocket` APIs.
```

Comment:
```suggestion
HTTP requests library for WASM Apps. It provides idiomatic Rust bindings for the `web_sys` `EventSource` (SSE), `fetch` and `WebSocket` APIs.
```

Also, it would be nice to add MDN links for `EventSource`, `fetch` and `WebSocket`

Reply:
Commit 2490c1ebe89440d0fc2748b330fc10eb3cce5ce3 addresses this.